setup.py

```diff
@@ -16,15 +16,15 @@
 
 setuptools.setup(
     name="comet-common",
-    version="3.0.0",
+    version="3.1.0",
```

Comment:
```suggestion
    version="3.1.0",
```
I think we should bump this to `4.0.0` since this includes breaking changes. Using this with the internal Spotify code base would cause issues if we don't update that Marshmallow code as well.